guides/common/assembly_provisioning-virtual-machines-on-a-kvm-server.adoc

```diff
@@ -0,0 +1,62 @@
+
+You can use KVM provisioning to create hosts over a network connection or from an existing image.
+
+.Prerequisites
+
+include::modules/snip_common-compute-resource-prereqs.adoc[]
+* A {SmartProxyServer} managing a network on the KVM server. Ensure no other DHCP services run on this network to avoid conflicts with the {SmartProxyServer}. For more information about network service configuration for {SmartProxyServer}s, see xref:Configuring_Networking[].
+ifeval::["{build}" == "satellite"]
+* A {RHEL} server running KVM virtualization tools (libvirt daemon). For more information, see the https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Virtualization_Getting_Started_Guide/index.html[Red Hat Enterprise Linux 7 Virtualization Getting Started Guide].
+endif::[]
+ifeval::["{build}" != "satellite"]
+* A server running KVM virtualization tools (libvirt daemon).
+* A virtual network running on the libvirt server. Only NAT and isolated virtual networks can be managed through {Project}.
+endif::[]
+* An existing virtual machine image if you want to use image-based provisioning. Ensure that this image exists in a storage pool on the KVM host. The `default` storage pool is usually located in `/var/lib/libvirt/images`. Only directory pool storage types can be managed through {Project}.
+* A {Project} user account with the following roles:
+** `Edit hosts`
+** `View hosts`
++
+For more information, see link:{BaseURL}administering_red_hat_satellite/chap-red_hat_satellite-administering_red_hat_satellite-users_and_roles#sect-{Project_Link}-Administering_{Project_Link}-Creating_and_Managing_Users-Assigning_Roles_to_a_User[Assigning Roles to a User] in the _Administering {ProjectName}_ guide.
+* A custom role in {Project} with the following permissions:
+** `view_compute_resources`
+** `destroy_compute_resources_vms`
+** `power_compute_resources_vms`
+** `create_compute_resources_vms`
+** `view_compute_resources_vms`
+** `view_locations`
+** `view_subnets`
++
+For more information about creating roles, see link:{BaseURL}administering_red_hat_satellite/chap-red_hat_satellite-Administering_{Project_Link}-Users_and_Roles#sect-red_hat_satellite-Administering_{Project_Link}-Creating_and_Managing_Roles-Creating_a_Role[Creating a Role] in the _Administering {ProjectName}_ guide. For more information about adding permissions to a role, see link:{BaseURL}administering_red_hat_satellite/chap-red_hat_satellite-Administering_{Project_Link}-Users_and_Roles#sect-red_hat_satellite-Administering_{Project_Link}-Creating_and_Managing_Roles-Adding_Permissions_to_a_Role[Adding Permissions to a Role] in the _Administering {ProjectName}_ guide.
+
+.Procedure Overview
+
+. xref:configuring-server-for-kvm-connections_kvm-provisioning[].
+. xref:adding-a-kvm-connection_kvm-provisioning[].
+. Optional: xref:adding-kvm-images_kvm-provisioning[]. Use this procedure if you want to use image-based provisioning.
```

Comment:
This xref does not render for some reason.

Reply:
Ah, I changed that file and forgot to update this xref. Thanks!